packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeDefinition.ts

```diff
@@ -5,7 +5,10 @@
 
 import { defer, from, map, merge, mergeAll, mergeMap } from "rxjs";
 import {
-  createNodesQueryClauseFactory, createPredicateBasedHierarchyDefinition, NodeSelectClauseColumnNames, ProcessedHierarchyNode,
+  createNodesQueryClauseFactory,
+  createPredicateBasedHierarchyDefinition,
+  NodeSelectClauseColumnNames,
+  ProcessedHierarchyNode,
 } from "@itwin/presentation-hierarchies";
```

Comment:
Why are there cases of imports becoming multiline without adding or changing the imports themselves?

Reply:
Prettier updated imports